package.json

```diff
@@ -36,7 +36,10 @@
     "modules/draft/esl-carousel/core/view/*.js",
     "src/modules/draft/esl-carousel/core/view/*.ts",
     "polyfills/**/*.js",
-    "src/polyfills/**/*.ts"
+    "src/polyfills/**/*.ts",
+    "pages/src/common/*.ts",
```

Comment:
You can basically move everything under the pages/src/**/*.ts under side effects to not overload library sideEffects with auxiliary paths.

Reply:
good point